src/components/App.tsx

```diff
@@ -0,0 +1,15 @@
+import MainScreen from '../pages/main';
+
+type AppProps = {
+  placeCardsCount: number;
+  cities: string[];
+  isNavigation: boolean;
+}
+
+function App(props: AppProps): JSX.Element {
+  return (
+    <MainScreen placeCardsCount={props.placeCardsCount} cities={props.cities} isNavigation={props.isNavigation} />
```

Comment:
Лучше деструктуизировать
```suggestion
function App({placeCardsCount, cities, isNavigation}: AppProps): JSX.Element {
  return (
    <MainScreen placeCardsCount={placeCardsCount} cities={cities} isNavigation={isNavigation} />
```